src/lib/relay/QueryRenderers.tsx

```diff
@@ -284,15 +284,16 @@ export const CollectionRenderer: React.SFC<CollectionRendererProps> = ({ collect
   <QueryRenderer<QueryRenderersCollectionQuery>
     environment={environment}
     query={graphql`
-      query QueryRenderersCollectionQuery($collectionID: String!, $screenWidth: Int) {
+      query QueryRenderersCollectionQuery($collectionID: String!, $screenWidth: Int, $headerHeight: Int) {
         collection: marketingCollection(slug: $collectionID) {
-          ...Collection_collection @arguments(screenWidth: $screenWidth)
+          ...Collection_collection @arguments(screenWidth: $screenWidth, headerHeight: $headerHeight)
         }
       }
     `}
     variables={{
       collectionID,
-      screenWidth: Dimensions.get("screen").width,
+      screenWidth: Dimensions.get("screen").width > 500 ? Dimensions.get("screen").width : 500,
+      headerHeight: Dimensions.get("screen").height > 700 ? Dimensions.get("screen").height : 204,
```

Comment:
The dimensions stuff is intentionally part of this PR, yeah?

Reply:
Yes, it was a small enough change to re-size the header in the same PR so I included it.